.flake8

```diff
@@ -2,4 +2,5 @@
 # https://github.com/PyCQA/flake8/issues/234
 [flake8]
 max-line-length = 120
-ignore = E123,E126,E127,E128,E129,E203,E231,E302,E303,E305,E501,E731,W291,W292,W293,W503,W504,DUO107,DUO104,DUO130,DUO109,DUO116
+# E501 doesn't work with black together
```

Comment:
@gruebel What do you mean by `black`?

Reply:
that's the one https://github.com/psf/black 🙂